cpu/Makefile.include.cortexm_common

```diff
@@ -58,13 +58,20 @@ export CFLAGS += -DCPU_ARCH_$(ARCH)
 
 # set the compiler specific CPU and FPU options
 ifeq ($(CPU_ARCH),cortex-m4f)
-# TODO: enable hard floating points for the M4F once the context save/restore
-#       code is adjusted to take care of FPU registers
-#export CFLAGS_FPU += -mfloat-abi=hard -mfpu=fpv4-sp-d16
-export MCPU := cortex-m4
-endif
+    USEMODULE += cortexm_fpu
+    ifneq (,$(filter cortexm_fpu,$(DISABLE_MODULE)))
+        export CFLAGS_FPU ?= -mfloat-abi=soft
+    else
+        # clang assumes there is an FPU
+        ifneq (llvm,$(TOOLCHAIN))
```

Comment:
Wihtout this ifneq, the following error is generated:
```
src/RIOT/cpu/cortexm_common/include/cmsis_gcc.h:344:54: error: unknown register name 'vfpcc' in asm
  __ASM volatile ("VMSR fpscr, %0" : : "r" (fpscr) : "vfpcc");
                                                     ^
1 error generated.
```

clang version : `3.8.1-12ubuntu1`